src/dictionaries.py

```diff
@@ -34,11 +34,19 @@
 ]
 
 # Add a new waypoint to the list
-# YOUR CODE HERE
+waypoints.append({
+    "lat": 23,
+    "lon": 33,
+    "name": "a fourth place"
+})
 
 # Modify the dictionary with name "a place" such that its longitude
 # value is -130 and change its name to "not a real place"
-# YOUR CODE HERE
+for waypoint in waypoints:
+    if waypoint.get('name') == 'a place':
+        waypoint["name"] = 'not a real place'
+        waypoint["lon"] = -130
```

Comment:
This is the more proper way to find the correct dict, as opposed to the way I mentioned in standup yesterday about just changing the properties of the first/zeroth dict.